python/paddle/jit/layer.py

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) 2019 PaddlePaddle Authors. All Rights Reserved.
+# Copyright (c) 2021 NVIDIA Corporation. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from paddle.fluid.core import Load
+
+
+class Layer():
+
+    def __init__(self):
+        self.cpp_layer = None
+        # {name: Function}
+        self.funciton_map = {}
+
+    def load(self, load_path, place):
+        self.cpp_layer = Load(load_path, place)
+        function_dict = self.cpp_layer.function_dict()
+
+        for name, function in function_dict.items():
+            self.funciton_map[name] = Function(function)
+            setattr(self, name, self.funciton_map[name].__call__)
```

Comment:
```suggestion
            setattr(self, name, self.funciton_map[name])
```
是不是不需要设置__call__，直接setattr对应的Function即可？